scripts/webpack.config.js

```diff
@@ -71,7 +70,6 @@ function getConfig(isReactExternalized) {
             alias: {
                 'box-ui-elements/es': path.join(__dirname, '../src'), // for examples only
                 examples: path.join(__dirname, '../examples/src'), // for examples only
-                'react-intl-locale-data': path.resolve(`node_modules/react-intl/locale-data/${locale}`),
```

Comment:
https://github.com/formatjs/react-intl/blob/master/docs/Upgrade-Guide.md#webpack might have to add to line 91